### PR TITLE
add learn more button hookup

### LINKED ISSUE #172: Hook up edit FAQ link at bottom of entry detail
When viewing an entry detail, I should be informed that if I wish to edit or delete this entry, it will need to happen via Firefox desktop or mobile.

## Acceptance criteria
* Should capture user intent to manage an entry (edit, delete)
* Should inform the user that management capabilities (edit, delete) are only available in the browser for now
* Should deep link to FAQ instruction for editing both in Firefox iOS and desktop Firefox

**NOTE** This is a change from the original design. The disclaimer needs to be revised

"Learn how to edit this entry" link directs to: https://lockbox.firefox.com/faq.html#how-do-i-edit-existing-entries

## Visual Design
**InVision**
https://mozilla.invisionapp.com/share/BQEUSO0S2#/267560603_Entry_Detail

**Zeplin**
https://zpl.io/VK4vXDX

Link: https://lockbox.firefox.com/faq.html#how-do-i-edit-existing-entries

depends on #44

### PR DESCRIPTION
**Pull request by sashei**
resolves #172

### CODE REVIEW

File: lockbox-ios/View/StaticURLWebView.swift [hunk 1]
```diff
@@ -38,5 +38,9 @@ class StaticURLWebView: UIViewController {
 
     private func styleNavigationBar() {
         self.navigationItem.title = self.navTitle
+
+        if #available(iOS 11.0, *) {
+            self.navigationItem.largeTitleDisplayMode = .never
```
Comment (devinreams) [inline lockbox-ios/View/StaticURLWebView.swift]: @sashei I hope you don't mind. I committed this here so the large title display from detail view didn't remain when loading the static URL view.

I was troubleshooting locally and had the change already in place so figured it was simpler for me to push a commit then ask you in a review. Feel free to back out if it's not the right result/approach though.

**Comment by changecourse** [comment]
What's going on with the back button copy @devinreams?

**Comment by sashei** [comment]
can fix back button copy; I imagine that's a UX rejection @changecourse

**Comment by devinreams** [comment]
@sashei what do you think about @changecourse's proposal to change this to the "slide up from bottom" view?

If the effort there is a bit more than "easy" I'd prefer we land the link as-is just so its in place and follow up later to improve the interaction. It's entirely up to you, though as I know this is an area ripe for consistency/refactoring.

**Comment by sashei** [comment]
@devinreams @changecourse err yes I've been exploring refactoring options this morning. They aren't hard but they're not easy either.

**Comment by nickbrandt** [comment]
@sashei I think this all looks great! 👍 I'm fine with losing the transition for the Settings screen, and think it works well with how we are opening web views now anyways.

File: lockbox-iosTests/RouteActionSpec.swift [hunk 2]
```diff
@@ -130,19 +124,33 @@ class RouteActionSpec: QuickSpec {
...
                   expect(LoginRouteAction.fxa.extras).to(beNil())
-                  expect(LoginRouteAction.learnMore.extras).to(beNil())
+                }
+            }
+        }
+
+        describe("ExternalWebsiteRouteAction") {
+            describe("equality") {
+                it("actions are equal when the title and URL string are equal") {
+                    expect(ExternalWebsiteRouteAction(urlString: "www.butts.com", title: "blah", returnRoute: MainRouteAction.list))
```
Comment (linuxwolf) [inline lockbox-iosTests/RouteActionSpec.swift]: 🤔 about the `urlString` ...

Comment (linuxwolf) [inline lockbox-iosTests/RouteActionSpec.swift]: I think a less ... controversial ... URL ought to be used herein.

Maybe "cats.example" or "cats.com"?

File: lockbox-ios/Action/RouteAction.swift [hunk 3]
```diff
@@ -19,10 +19,21 @@ class RouteActionHandler: ActionHandler {
 
 protocol RouteAction: Action { }
 
+struct ExternalWebsiteRouteAction: RouteAction {
+    let urlString: String
+    let title: String
+    let returnRoute: RouteAction
```
Comment (jhugman) [inline lockbox-ios/Action/RouteAction.swift]: 😍

File: lockbox-ios/Common/Resources/Constants.swift [hunk 4]
```diff
@@ -10,9 +10,11 @@ struct Constant {
     struct app {
         static let redirectURI = "https://mozilla-lockbox.github.io/fxa/ios-redirect.html"
         static let faqURL = "https://lockbox.firefox.com/faq.html"
+        static let privacyURL = "https://lockbox.firefox.com/privacy.html"
         static let provideFeedbackURL = "https://qsurvey.mozilla.com/s3/Lockbox-Input"
-        static let useLockboxFAQ = "https://lockbox.firefox.com/faq.html#how-do-i-use-lockbox"
+        static let useLockboxFAQ = "https://lockbox.firefox.com/faq.html#how-do-i-use-firefox-lockbox"
```
Comment (jhugman) [inline lockbox-ios/Common/Resources/Constants.swift]: 🙀

File: lockbox-ios/Presenter/RootPresenter.swift [hunk 5]
```diff
@@ -169,33 +170,48 @@ class RootPresenter {
...
-            case .faq:
-                view.pushSettingView(view: .faq)
-            case .provideFeedback:
-                view.pushSettingView(view: .provideFeedback)
             }
+        }.asObserver()
+    }()
 
+    lazy private var showExternalWebsite: AnyObserver<ExternalWebsiteRouteAction> = { [unowned self] in
+        return Binder(self) { target, externalSiteAction in
```
Comment (jhugman) [inline lockbox-ios/Presenter/RootPresenter.swift]: ❓ 🆗

File: lockbox-iosTests/AutoLockStoreSpec.swift [hunk 6]
```diff
@@ -211,7 +211,7 @@ class AutoLockStoreSpec: QuickSpec {
                     }
 
                     itBehavesLike(AutoLockStoreSpecSharedExample.TimerReset.rawValue) {
-                        [AutoLockStoreSpecContext.Action.rawValue: ExternalLinkAction(baseURLString: "www.butts.com")]
+                        [AutoLockStoreSpecContext.Action.rawValue: ExternalLinkAction(baseURLString: "www.example.com")]
```
Comment (jhugman) [inline lockbox-iosTests/AutoLockStoreSpec.swift]: 🤔

**Pull request merged**